docs/confounding-variables.md

```diff
@@ -0,0 +1,40 @@
+# Confounding Variables
+
+A confounding variable is a factor that influences independent variable and dependant variable. 
+A confounding variable can mix with the effect of an independent variable on the dependent variable thus leading to erronous conclusions about the experiment.
+
+At any time during your experiment, the data manager allows you to capture confounding variables. 
+
+### Define confounding variables in your experiment
+
+In order to work with confounding variables, first you must define them in your experiment. 
+Confounding variables are experiment specific. Confounding variables do not interact with
```

Comment:
The scope of confounding variables in Data Manager are ....

Reply:
_The scope of a confounding variable is bound to an experiment in Data Manager._ 

Or: 

_Confounding variables are experiment-specific. Currently, it is not possible to reuse the same variable in other experiments metadata in Data Manager._

 > Confounding variables do not interact with
information outside your experiment.

I don't understand this statement.